_drafts/SC1-2-2-video-captions.md

```diff
@@ -0,0 +1,98 @@
+---
+name: video captions
+description: |
+  Captions are available for audio information in non-streaming `video` elements.
+
+success_criterion:
+- 1.2.2 # Captions (Prerecorded)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- [Audio output]{#audio-output}
+- [Visual output]{#audio-output}
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video contains [audio](#audio).
+
+### Expectation
+
+For each test target, audio information that is not expressed visually in the video, is available through [captions](#captions).
+
+*Note*: Captions can be either embedded in the video file itself or can be made available trough a separate track.
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-captions.html
+- https://www.w3.org/TR/WCAG20-TECHS/G93.html
+- https://www.w3.org/TR/WCAG20-TECHS/G87.html
+- https://www.w3.org/TR/WCAG20-TECHS/H95.html
+
+## Test Cases
+
+### Passed
+
+```html
```

Comment:
add comments/ description for each test case

Reply:
We have added comments to all the test cases. Thank you for the feedback.